### ISSUE TITLE
FreeBSD 13.1-RELEASE

### ISSUE THREAD

**Issue by kaiquekandykoga** [tj/n#744]
# Bug Report

## Summary

I was not being able to install n using some of the methods described in the README.

method 0

```
curl -L https://raw.githubusercontent.com/tj/n/master/bin/n -o n
bash n lts
```

method 1

`make install`

_I was only able to install it using npm install -g n._

Once installed I have tried to install different versions of Node.js, but I did not have any success.

## Steps to Reproduce

```
[kandy@ ~/a]$ n lts

  Error: no version found for 'lts'
```

```
[kandy@ ~/a]$ n i 18.12.0
  installing : node-v18.12.0
curl: (22) The requested URL returned error: 404

  Error: download preflight failed for '18.12.0' (https://nodejs.org/dist/v18.12.0/node-v18.12.0-unexpected_os-amd64.tar.gz)
```

```
[kandy@ ~/a]$ n ls-remote
Listing remote... Displaying 20 matches (use --all to see all).
```

_Nothing is shown using the ls-remote command._

## Expected Behaviour

Being able to install Node.js using n on FreeBSD.

## Actual Behaviour

Not being able to install Node.js using n on FreeBSD.

## Other Information

I am not sure if FreeBSD is a supported system for n. If it is not, please feel free to close the issue. Thank you in advance.

```
[kandy@ ~/a]$ freebsd-version
13.1-RELEASE
```

## Configuration Details

```bash
[kandy@ ~/a]$ n --version
v9.0.0

[kandy@ ~/a]$ node -v
v16.17.1

[kandy@ ~/a]$ node -p process.platform
freebsd
```

**Comment by shadowspawn** [comment]
Thanks for the detail.

`n` uses prebuilt binaries available from Node.js, and does not build from source. FreeBSD is not currently supportedl:

- https://github.com/nodejs/build/issues/2965
- https://github.com/nodejs/node/blob/main/BUILDING.md#platform-list

(We could look into the install problems for `n` itself as I am not sure why they didn't work, but probably not of interest to you since `n` is not able to find FreeBSD versions to download!)

**Comment by shadowspawn** [comment]
`Makefile` improvements released in `n` v9.0.1.

**Comment by shadowspawn** [comment]
FreeBSD issue explained, and `make` issue identified and fixed. No further activity in a month. Closing this as resolved.

Thank you  for your contributions.

Feel free to open a new issue if it comes up again, with new information and renewed interest.
